docs/getting-started.md

```diff
@@ -0,0 +1,179 @@
+# Getting Started
+
+## Installation
+
+1. Clone the Regula repo and move to the new directory:
+
+        git clone https://github.com/fugue/regula.git
```

Comment:
This is going to change soon, but maybe we should recommend that they clone at a specific tag since these instructions will fall out of date for a few days:

```shell
git clone -b v0.8.0 https://github.com/fugue/regula.git
```